### PR TITLE
Added URI linking

### PR DESCRIPTION
**Pull request by JohnMulligan**
Uses same structure as addLink

addURI(self, pagenum, uri, rect, border=None)

### CODE REVIEW

**Comment by oscardssmith** [comment]
pretty sure this should be closed now

File: PyPDF2/pdf.py [hunk 1]
```diff
@@ -895,6 +895,73 @@ def removeText(self, ignoreByteStringObject=False):
...
+			pass
+		else:
+			rect = RectangleObject(rect)
+
+		lnk2 = DictionaryObject()
+
+		lnk2.update({
+		NameObject('/S'): NameObject('/URI'),
+		NameObject('/URI'): TextStringObject(uri)
+		});
```
Comment (soferio) [inline PyPDF2/pdf.py]: Semicolon here probably unnecessary. Works very nicely though!

**Comment by reteps** [comment]
Add please!

**Comment by MartinThoma** [comment]
I'm open to adding this, but the PR needs to be updated. It might be simpler to create a new PR.

**Comment by MartinThoma** [comment]
Thank you for this PR!

A very similar one was merged a while ago: https://github.com/py-pdf/PyPDF2/pull/274

Hence this one is no longer necessary.

**Pull request closed without merging**